met-api/src/met_api/resources/event_item_translation.py

```diff
@@ -0,0 +1,152 @@
+from met_api.utils.util import allowedorigins, cors_preflight
+
+
+API = Namespace(
+    'event_item_translations',
+    description='Endpoints for EventItemTranslation Management',
+)
+
+
+@cors_preflight('GET, POST, PATCH, DELETE, OPTIONS')
+@API.route('/<int:event_item_translation_id>')
+class EventItemTranslationResource(Resource):
+    """Resource for managing event item translations."""
+
+    @staticmethod
+    @cross_origin(origins=allowedorigins())
+    def get(event_id, event_item_translation_id):
+        """Fetch an event item translation by id."""
+        try:
+            event_item_translation = (
+                EventItemTranslationService.get_by_id(
+                    event_item_translation_id
+                )
+            )
+            return (
+                EventItemTranslationSchema().dump(event_item_translation),
+                HTTPStatus.OK,
+            )
+        except (KeyError, ValueError) as err:
+            return str(err), HTTPStatus.INTERNAL_SERVER_ERROR
+
+    @staticmethod
+    @_jwt.requires_auth
+    @cross_origin(origins=allowedorigins())
+    def patch(event_id, event_item_translation_id):
+        """Update saved event item translation partially."""
+        try:
+            request_json = request.get_json()
+            event_item_translation = (
+                EventItemTranslationService.update_event_item_translation(
+                    event_id, event_item_translation_id, request_json
+                )
+            )
+            return (
+                EventItemTranslationSchema().dump(event_item_translation),
+                HTTPStatus.OK,
+            )
+        except ValueError as err:
+            return str(err), HTTPStatus.NOT_FOUND
+        except ValidationError as err:
+            return str(err.messages), HTTPStatus.BAD_REQUEST
+
+    @staticmethod
+    @_jwt.requires_auth
+    @cross_origin(origins=allowedorigins())
+    def delete(event_id, event_item_translation_id):
+        """Delete an event item translation."""
+        try:
+            success = EventItemTranslationService.delete_event_item_translation(
+                event_id, event_item_translation_id
+            )
+            if success:
+                return (
+                    'Successfully deleted event item translation',
+                    HTTPStatus.NO_CONTENT,
+                )
+            raise ValueError('Event item translation not found')
+        except KeyError as err:
+            return str(err), HTTPStatus.BAD_REQUEST
+        except ValueError as err:
+            return str(err), HTTPStatus.NOT_FOUND
+
+
+@cors_preflight('GET, OPTIONS')
+@API.route('/item/<int:event_item_id>/language/<int:language_id>')
+class EventItemTranslationResourceByLanguage(Resource):
+    """Resource for event item using language_id."""
+
+    @staticmethod
+    @cross_origin(origins=allowedorigins())
+    def get(event_id, event_item_id, language_id):
```

Comment:
Not needed, but we need to mention it there otherwise flask route will throw error, so in next  PR I will change that to 

def get(event_item_id, language_id, **_):
where **_ is often used as a placeholder for a keyword argument in a function definition.